components/clp-package-utils/clp_package_utils/scripts/stop_clp.py

```diff
@@ -38,14 +39,24 @@
 logger.addHandler(logging_console_handler)
 
 
-def stop_container(container_name: str):
-    if not container_exists(container_name):
-        return
+def stop_running_container(container_name: str, exited_container: list, force: bool):
```

Comment:
`exited_container` -> `already_exited_containers`

Reply:
@haiqi96 I used the plural above (container**s**). Any reason why you used the singular in the code?

Reply:
sorry, fixed now